Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM rockylinux:8.9
+
+COPY docker/install.sh .
+COPY docker/entrypoint.sh .
```

Comment:
After thinking about this a little more - it would be good for the Dockerfile to just have the line

`COPY . /atchem`

So then it no longer would need to use curl. Right now though I am running into a dependency issue when trying to do this with the existing state of the master branch. I'm going to push some commits addressing this review first, but then I will try to work ontop of  1.2.2 tag and see if I can get this working

Reply:
Managed to get this working - so it should be able to run on any version of the repo - so to build the 1.2.2 image you can checkout that branch and build it (in theory). For future releases that should be able to be triggered by a Github action without changes to the Dockerfile - you'll just need to tag the build with the same tag as the release